xbmc/music/MusicUtils.cpp

```diff
@@ -718,27 +718,30 @@ bool IsItemPlayable(const CFileItem& item)
   if (item.IsPVR() || item.IsPlugin() || item.IsScript() || item.IsAddonsPath())
     return false;
 
-  // Check for right window
-  const int winID = CServiceBroker::GetGUI()->GetWindowManager().GetActiveWindow();
-  if (winID != WINDOW_MUSIC_NAV && winID != WINDOW_HOME)
-    return false;
-
-  if (item.m_bIsFolder)
+  if (item.m_bIsFolder &&
+      (item.IsMusicDb() || StringUtils::StartsWithNoCase(item.GetPath(), "library://music/")))
```

Comment:
This allows <play>, <play next> and <queue> to be displayed in the context menu in the playlist editor if the item is in the music db.  However, none of those options make much sense in that context - The items in there are already queued/played or currently playing.  Or at least, those options in that context do not make much sense to me :)

Reply:
I do agree and will have a look.

Reply:
@the-black-eagle was your comment based on a runtime-test? I'm not able to reproduce. Looks like this for me:

<img width="1660" alt="Screenshot 2022-11-05 at 17 09 57" src="https://user-images.githubusercontent.com/3226626/200129320-587d1e72-5021-4a47-9c29-a9a70798702c.png">

Reply:
Ah, we are looking at slightly different windows!  I played an album, went to the playlist and then fired up the context menu.  I guess `playlist editor` was a misnomer and I should have said `playlist`.
<a href="https://ibb.co/yRV8SbL"><img src="https://i.ibb.co/sPtFHML/screenshot00041.png" alt="screenshot00041" border="0"></a><br /><br />

Note that the top three context menu items are not available in v19.x .

Reply:
@the-black-eagle should be fixed with latest forced-push. Could you please retest?

Reply:
LGTM now.  Thanks !!